assets/base.css

```diff
@@ -155,9 +155,8 @@
   --color-icon: rgb(var(--color-base-outline-button-labels));
 }
 
-.product-grid,
-.collection-list,
-.blog__posts,
+.contains-card--standard,
+.contains-card--card,
```

Comment:
Since you only have modifiers atm, I would suggest applying those variables to `.contains-card` and add that class where you've added `.contains-card--style`.

Reply:
I went back and forth so much on these. I ultimately didn't add it because it wouldn't be useful, as used currently. But it theoretically makes sense. I may opt to just wait til we're making more use of these contexts (grid gap changes and make sure I'm still going to need them the way I think.

Reply:
I think you could have `.contains-card` and `.contains-card--standard` 🤔  I think it'd be nice to follow BEM

Reply:
Oh what I meant is maybe `contains-card` could be the default using the current styles from `contains-card--card`. And then only have one "modifier" with `--standard`. What do you think 🤔

Reply:
Ahh, I think I like that actually. Does this make sense? https://github.com/Shopify/dawn/pull/1292/commits/d7f333e89e2157cbf8d01a218e0f2cc45d281777

Reply:
Yeah I think so 👍